test/test_edit_all_mountings_and_quotas_backend.py

```diff
@@ -0,0 +1,340 @@
+        model.delete_all_tenta_mountings(self.DUMMY_MODULE_CODE_2)
+        model.delete_all_fixed_mountings(self.DUMMY_MODULE_CODE_3)
+        model.delete_all_tenta_mountings(self.DUMMY_MODULE_CODE_3)
+
+        model.delete_module(self.DUMMY_MODULE_CODE_1)
+        model.delete_module(self.DUMMY_MODULE_CODE_2)
+        model.delete_module(self.DUMMY_MODULE_CODE_3)
+
+
+    def test_edit_single_module_quota(self):
+        '''
+            Test that Edit All function can edit a single module's quota
+        '''
+        test_module_code = self.DUMMY_MODULE_CODE_1
+        test_mounting = True
+        test_quota = 100
+
+        # Modifiy Sem 1 quota
+        test_data = {}
+        test_data[test_module_code+'_isEdited'] = "True"
+        test_data[test_module_code+'_Sem1Mounting'] = test_mounting
+        test_data[test_module_code+'_Sem1Quota'] = test_quota  #modified
+
+        self.edit_all_handler.POST(test_data)
+
+        mounting = model.get_mounting_of_target_tenta_ay_sem(test_module_code,
+                                                             self.next_ay+" Sem 1")
+        quota = model.get_quota_of_target_tenta_ay_sem(test_module_code,
+                                                       self.next_ay+" Sem 1")
+        assert_equal(mounting, test_mounting)
+        assert_equal(quota, test_quota)
+
+        model.update_quota(test_module_code, self.next_ay+" Sem 1", self.DUMMY_QUOTA_1)
+
+        # Modify Sem 2 quota
+        test_module_code = self.DUMMY_MODULE_CODE_2
+        test_mounting = True
+        test_quota = 200
+
+        test_data = {}
+        test_data[test_module_code+'_isEdited'] = "True"
+        test_data[test_module_code+'_Sem2Mounting'] = test_mounting
+        test_data[test_module_code+'_Sem2Quota'] = test_quota  #modified
+
+        self.edit_all_handler.POST(test_data)
+
+        mounting = model.get_mounting_of_target_tenta_ay_sem(test_module_code,
+                                                             self.next_ay+" Sem 2")
+        quota = model.get_quota_of_target_tenta_ay_sem(test_module_code,
+                                                       self.next_ay+" Sem 2")
+        assert_equal(mounting, test_mounting)
+        assert_equal(quota, test_quota)
+
+        model.update_quota(test_module_code, self.next_ay+" Sem 2", self.DUMMY_QUOTA_2)
+
+
+    def test_edit_single_module_multiple_quotas(self):
+        '''
+            Test that Edit All function can edit a single module's multiple quotas
+        '''
+        test_module_code = self.DUMMY_MODULE_CODE_3
+        test_mounting_1 = True
+        test_mounting_2 = True
+        test_quota_1 = None
+        test_quota_2 = 300
+
+        # Modify Sem 1 and 2 quotas
+        test_data = {}
+        test_data[test_module_code+'_isEdited'] = "True"
+        test_data[test_module_code+'_Sem1Mounting'] = test_mounting_1
+        #test_data[test_module_code+'_Sem1Quota'] = test_quota_1   #modified to empty
```

Comment:
There are other instances of commented-out lines in this suite of test cases. Seeing they are commented out, why are they made so? Are those needed or not needed? If needed, please make it clear on why these commented-out lines should be there. If not, please remove them.

If not commenting these lines *do* actually break something, please fix the bugs.